backend.native/tests/interop/objc/tests/tryRetainGC.kt

```diff
@@ -0,0 +1,25 @@
+import kotlinx.cinterop.*
+import kotlin.test.*
+import objcTests.*
+
+@Test
```

Comment:
I don't see this test added to the `backend.native/tests/build.gradle`. Is it intentional?

Reply:
It is not required for `backend.native/tests/interop/objc/tests/*`

Reply:
Ah, yes. I see.